### PR TITLE
separated styles.css into separate files, created toast.css to fix du…

### PR DESCRIPTION
**Pull request by shivasaurus**
connect #164

### CODE REVIEW

**Review by rmroot — CHANGES_REQUESTED** [review]
We should place all of these files in a styles folder instead of just src

**Comment by shivasaurus** [comment]
CSS files have been placed into a styles directory

**Review by cranfillkd — APPROVED** [review]

File: src/app/electron/electron-update/electron-update.component.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ import { LoadingService } from 'src/app/shared/loading/loading.service';
 @Component({
   selector: 'app-electron-update',
   templateUrl: './electron-update.component.html',
-  styleUrls: ['./electron-update.component.css'],
+  styleUrls: ['../../../styles/toast.css'],
```
Comment (rmroot) [inline src/app/electron/electron-update/electron-update.component.ts:10]: In here and the toast component we *do not* want to update the styleUrls. The toast.css file should be a global style sheet like the others. I think it still needs to be loaded in angular.json

File: angular.json [hunk 2]
```diff
@@ -23,7 +23,24 @@
               "src/assets"
             ],
             "styles": [
-              "src/styles.css",
+              "src/styles/styles.css",
+              "src/styles/pagination.css",
+              "src/styles/slidertoggle.css",
```
Comment (rmroot) [inline angular.json:28]: let's use camel casing to name these files. "sliderToggle.css" "profilePicture.css" etc..

File: src/app/electron/electron-update/electron-update.component.ts (hunk 1 above)
Comment (rmroot) [reply src/app/electron/electron-update/electron-update.component.ts:10]: If that doesn't work, let me know and I can dig in a bit to what's going on

**Review by rmroot — CHANGES_REQUESTED** [review]

**Review by rmroot — APPROVED** [review]

**Pull request merged**